src/modules/dashboard/application/controllers/__tests__/dashboard-layout.controller.spec.ts

```
@@ -0,0 +1,298 @@
+  GetLayoutUseCase,
+  ListLayoutsUseCase,
+  SetDefaultLayoutUseCase,
+} from '../../use-cases/layouts';
+import { JwtPayload } from '@/core/types/jwt-payload.interface';
+import {
+  CreateDashboardLayoutDto,
+  UpdateDashboardLayoutDto,
+  DashboardLayoutResponseDto,
+  DashboardLayoutListResponseDto,
+} from '../../dto/dashboard-layout.dto';
+import { DashboardLayout } from '@/modules/dashboard/domain/entities/dashboard-layout.entity';
```

ESLint

[error] 'DashboardLayout' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)